web-common/src/features/components/charts/funnel/spec.ts

```diff
@@ -286,6 +287,16 @@ export function generateVLFunnelChartSpec(
     },
   };
 
+  // Resolve CSS variables for canvas rendering
+  const darkTextColor = resolveCSSVariable(
+    "var(--color-gray-900)",
+    data.isDarkMode,
+  );
+  const lightTextColor = resolveCSSVariable(
+    "var(--color-gray-50)",
+    data.isDarkMode,
```

Comment:
Should the CSS resolve care about if theme is in dark mode or light mode in this case? The dark text color should always be dark and light text color always light.